rosbridge_library/src/rosbridge_library/protocol.py

```diff
@@ -222,8 +235,7 @@ def incoming(self, message_string: str = "") -> None:
         # This way, a client can change/overwrite its active values anytime by just including parameter field in any
         # message sent to rosbridge. Maybe need to be improved to bind parameter values to specific operation.
         if "fragment_size" in msg:
-            self.fragment_size = msg["fragment_size"]
-            # print "fragment size set to:", self.fragment_size
+            self.fragment_size = min(msg["fragment_size"], self.max_message_size)
```

Comment:
What happens when a message has more data than the max message size?

Reply:
It gets fragmented into multiple messages